mozecode/dose
# Gm style

I made the following changes:
- basic styling with navbar, welcome
- add styles for welcome page with font awesome icons
- add brand image to navbar
- add background image and tweak styles
- add styles for add Prescription form
- add styles for updateScript view
- add create prescription api call result styling
- position radio buttons for add prescription
- styles for registration form
- add styles for account details view
- add button styles
- wrap up all styles except for see all prescriptions view

To test:  
Use the google dev tools to look at the mobile, tablet, and desktop responsive versions of the site.  This site should be mobile, tablet, and desktop friendly with the exception of the see all prescriptions view at this point.
